io/disk/sysfs_att.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Maram Srimannarayana Murthy <Maram.Srimannarayana.Murthy@ibm.com>
+
+"""
+Check all sysfs queue attributes by reading or writing different values
+"""
+
+import os
+from avocado import Test
+from avocado.utils import genio
+
+
+class Sysfs_Queue(Test):
+
+    """
+    Checks all Sysfs_Queue attributes
+    """
+
+    def setUp(self):
+        """
+        Reading required arguments from yaml file
+        Setting up file paths
+        """
+        self.disk = self.params.get("disk", default="")
+        self.queue_param_disk_dir = os.path.join("/sys/block",
+                                                 self.disk, "queue")
+        self.param_name = self.params.get("param_name",
+                                          default="None")
+        self.attribute_values = self.params.get("values").split(" ")
+        if self.disk == "":
```

Comment:
Please move this if block just after self.params.get(disk) statement so that we can avoid failure in subsequent statements if no value for disk is provided.